dex/src/main/scala/com/wavesplatform/dex/cli/WavesDexCli.scala

```diff
@@ -1,43 +1,43 @@
 package com.wavesplatform.dex.cli
 
 import cats.Id
-import cats.syntax.option._
-import cats.syntax.either._
 import cats.instances.either._
+import cats.syntax.either._
+import cats.syntax.option._
 import com.typesafe.config.Config
 import com.typesafe.config.ConfigFactory.parseFile
 import com.wavesplatform.dex._
 import com.wavesplatform.dex.app.{forceStopApplication, MatcherStateCheckingFailedError}
-import com.wavesplatform.dex.db.{AccountStorage, DbKeys}
 import com.wavesplatform.dex.db.leveldb.{openDb, LevelDb}
+import com.wavesplatform.dex.db.{AccountStorage, DbKeys}
 import com.wavesplatform.dex.doc.MatcherErrorDoc
 import com.wavesplatform.dex.domain.account.{AddressScheme, KeyPair}
 import com.wavesplatform.dex.domain.bytes.ByteStr
 import com.wavesplatform.dex.domain.bytes.codec.Base58
 import com.wavesplatform.dex.error.Implicits.ThrowableOps
-import com.wavesplatform.dex.settings.{loadConfig, loadMatcherSettings, MatcherSettings}
-import com.wavesplatform.dex.tool.connectors.SuperConnector
+import com.wavesplatform.dex.settings.{loadMatcherSettings, MatcherSettings}
 import com.wavesplatform.dex.tool._
+import com.wavesplatform.dex.tool.connectors.SuperConnector
 import monix.eval.Task
-import monix.execution.{ExecutionModel, Scheduler}
 import monix.execution.schedulers.SchedulerService
-import pureconfig.ConfigSource
+import monix.execution.{ExecutionModel, Scheduler}
 import scopt.{OParser, RenderingMode}
 import sttp.client3._
 
 import java.io.{File, PrintWriter}
 import java.nio.charset.StandardCharsets
 import java.nio.file.Files
-import java.util.{Base64, Scanner}
 import java.util.concurrent.atomic.AtomicLong
-import scala.concurrent.{Await, TimeoutException}
+import java.util.{Base64, Scanner}
 import scala.concurrent.duration.{DurationInt, FiniteDuration}
+import scala.concurrent.{Await, TimeoutException}
 import scala.util.{Failure, Success, Try, Using}
 
 object WavesDexCli extends ScoptImplicits {
 
   private val backend = HttpURLConnectionBackend()
 
+  // noinspection ScalaStyle
```

Comment:
Wondering, why do we need this?

Reply:
the function below uses ```println``` which will be falsely highlighted by Idea